### DISCUSSION TITLE
When can I get the verison 3.1.5

### DISCUSSION THREAD

**Discussion by nickychen123456** [openssl/openssl#22882] [Q&A] [answered]
We need to upgrade the openssl verison because of the follow CVEs:

[CVE-2023-5363](https://www.cve.org/CVERecord?id=CVE-2023-5363)

[CVE-2023-2975](https://blackduck.corp.zoom.us/api/vulnerabilities/CVE-2023-2975)

[CVE-2023-3817](https://blackduck.corp.zoom.us/api/vulnerabilities/CVE-2023-3817)

[CVE-2023-5678](https://www.openssl.org/news/vulnerabilities.html#CVE-2023-5678)

The CVE-2023-5678 was fixed in 3.1.5 version according to the release notes, but I donot see the version 3.1.5 in the release list, So where can I get it?

**Comment by t8m** [comment, marked as answer]
As mentioned in https://www.openssl.org/news/secadv/20231106.txt the issue is of Low severity which means we are not releasing the 3.1.5 version at the time of the disclosure. We will release 3.1.5 later.
